configuration/developing-community-health-applications.md

```diff
@@ -557,13 +557,13 @@ An array of fields are defined to show a summary of the contact at the top of th
 
 Each field that can be shown on a contact's profile is defined as an object in the `fields` array. The properties for each object determine how and when the field is shown.
 
+<!-- If you change this table, update the duplicate descriptions in ### Cards -->
```

Comment:
You moved this comment, but didn't read it!

Reply:
Lol. Yup, sounds about right.